helm/vitess/templates/_orchestrator-conf.tpl

```diff
@@ -0,0 +1,145 @@
+    "MySQLTopologyCredentialsConfigFile": "",
+    "MySQLTopologyMaxPoolConnections": 3,
+    "MySQLTopologyPassword": "orc_client_user_password",
+    "MySQLTopologyReadTimeoutSeconds": 3,
+    "MySQLTopologySSLCAFile": "",
+    "MySQLTopologySSLCertFile": "",
+    "MySQLTopologySSLPrivateKeyFile": "",
+    "MySQLTopologySSLSkipVerify": true,
+    "MySQLTopologyUseMutualTLS": false,
+    "MySQLTopologyUser": "orc_client_user",
+    "OnFailureDetectionProcesses": [
+        "echo 'Detected {failureType} on {failureCluster}. Affected replicas: {countSlaves}' >> /tmp/recovery.log"
+    ],
+    "OSCIgnoreHostnameFilters": [
+    ],
+    "PhysicalEnvironmentPattern": "[.]([^.]+[.][^.]+)[.]mydomain[.]com",
+    "PostFailoverProcesses": [
+        "echo '(for all types) Recovered from {failureType} on {failureCluster}. Failed: {failedHost}:{failedPort}; Successor: {successorHost}:{successorPort}' >> /tmp/recovery.log"
+    ],
+    "PostIntermediateMasterFailoverProcesses": [
+        "echo 'Recovered from {failureType} on {failureCluster}. Failed: {failedHost}:{failedPort}; Successor: {successorHost}:{successorPort}' >> /tmp/recovery.log"
+    ],
+    "PostMasterFailoverProcesses": [
+        "echo 'Recovered from {failureType} on {failureCluster}. Failed: {failedHost}:{failedPort}; Promoted: {successorHost}:{successorPort}' >> /tmp/recovery.log",
+        "vtctlclient -server vtctld.{{ $namespace }}:15999 TabletExternallyReparented {successorAlias}"
+    ],
+    "PostponeSlaveRecoveryOnLagMinutes": 0,
+    "PostUnsuccessfulFailoverProcesses": [
+    ],
+    "PowerAuthUsers": [
+        "*"
+    ],
+    "PreFailoverProcesses": [
+        "echo 'Will recover from {failureType} on {failureCluster}' >> /tmp/recovery.log"
+    ],
+    "ProblemIgnoreHostnameFilters": [
+    ],
+    "PromotionIgnoreHostnameFilters": [
+    ],
+    "PseudoGTIDMonotonicHint": "asc:",
+    "PseudoGTIDPattern": "drop view if exists .*?`_pseudo_gtid_hint__",
+    "RaftAdvertise": "POD_NAME.{{ $namespace }}",
+    "RaftBind": "POD_NAME",
+    "RaftDataDir": "/var/lib/orchestrator",
+    "RaftEnabled": true,
+    "RaftNodes": [
+    {{ range $i := until (int $orc.replicas) }}
+        "orchestrator-{{ $i }}.{{ $namespace }}"{{ if lt $i (sub (int64 $orc.replicas) 1) }},{{ end }}
+    {{ end }}
+    ],
+    "ReadLongRunningQueries": false,
+    "ReadOnly": false,
+    "ReasonableMaintenanceReplicationLagSeconds": 20,
+    "ReasonableReplicationLagSeconds": 10,
+    "RecoverIntermediateMasterClusterFilters": [
+        "*"
+    ],
+    "RecoverMasterClusterFilters": [
+        ".*"
+    ],
+    "RecoveryIgnoreHostnameFilters": [
+    ],
+    "RecoveryPeriodBlockSeconds": 60,
+    "ReduceReplicationAnalysisCount": true,
+    "RejectHostnameResolvePattern": "",
+    "RemoveTextFromHostnameDisplay": ".mydomain.com:3306",
+    "ReplicationLagQuery": "SELECT unix_timestamp() - floor(ts/1000000000) FROM `_vt`.heartbeat WHERE keyspaceShard='KEYSPACE:SHARD';",
```

Comment:
@bbeaudreault This was a naive copy of your heartbeat query. I looked through your PR code to see if `KEYSPACE:SHARD` is being interpolated for this `SELECT` query or if the standard query parser does that, but I couldn't tell for sure.

Reply:
I think to query in that form, you'd need to know the keyspace and shard (which we don't here). As far as I can tell, there should still only be 1 row in any given heartbeat table. Maybe we can just leave off the `WHERE` clause and add `LIMIT 1`?